### ISSUE TITLE
Unable to run 'fornax build' or 'fornax watch' - seemingly due to error "Could not locate the assembly "FSharp.Core..."

### ISSUE THREAD

**Issue by drk-mtr** [LambdaFactory/Fornax#5]
Firstly, huge thanks for sharing your software in the public domain - incredibly generous.

Unfortunately I was unable to get this working on a Windows PC or a Mac. I suspect this may be due to it not being able to find the correct F# version, but I'm too new to coding to know whether this is actually the case.

Here are the errors I get with a 'build':

```
C:\Projects\FornaxTrial
> fornax build
Error: System.Exception: Error creating evaluation session: StopProcessingExn null
   at Microsoft.FSharp.Compiler.Interactive.Shell.-ctor@2571-142.Invoke(String message)
   at Microsoft.FSharp.Compiler.Interactive.Shell.FsiEvaluationSession..ctor(FsiEvaluationSessionHostConfig fsi, String[] argv, TextReader inReader, TextWriter outWriter, TextWriter errorWriter, Boolean fsiCollectible, Boolean msbuildEnabled)
   at Microsoft.FSharp.Compiler.Interactive.Shell.FsiEvaluationSession.Create(FsiEvaluationSessionHostConfig fsiConfig, String[] argv, TextReader inReader, TextWriter outWriter, TextWriter errorWriter, FSharpOption`1 collectible, FSharpOption`1 msbuildEnabled)
   at <StartupCode$Fornax>.$Generator..cctor()
Inner: <null>
ErrorStream:

error FS0082: Could not resolve this reference. Could not locate the assembly "FSharp.Core, Version=4.4.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a". Check to make sure the assembly exists on disk. If this reference is required by your code, you may get compilation errors. (Code=MSB3245)


error FS0073: internal error: BuildFrameworkTcImports: no resolution of 'FSharp.Core, Version=4.4.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'


Unhandled Exception: System.TypeInitializationException: The type initializer for '<StartupCode$Fornax>.$Generator' threw an exception. ---> System.Exception: Error creating evaluation session: StopProcessingExn null
   at <StartupCode$Fornax>.$Generator..cctor()
   --- End of inner exception stack trace ---
   at Generator.f@1(String n)
   at Generator.getPosts(String projectRoot)
   at Generator.generateFolder(String projectRoot)
   at Fornax.main$cont@57(String[] argv, ArgumentParser`1 parser, Unit unitVar)
   at Fornax.main(String[] argv)
```

The error is essentially the same for a 'watch'.

As I say, this may well be me misunderstanding where F# should be installed (in which case I apologise for recording this as an issue with Fornax!), however I can successfully run F# code on both computers and can use FSI successfully from the command line.

On the Windows PC, I have the following folders on this PC which suggest F# is installed:

`C:\Program Files (x86)\Microsoft SDKs\F#\4.1`
`C:\Program Files (x86)\Microsoft SDKs\F#\10.1`

Under both of these there is a 'Framework\v4.0' folder, the contents appear to be the same in each.

When starting FSI on the PC I get:

`Microsoft (R) F# Interactive version 4.1`

I have various dotnet framework version installed (including 4.6.2 and 4.7) and plenty of dotnet core versions - please let me know if you need any further info in this respect.

**Comment by Krzysztof-Cieslak** [comment]
Probably updating `FSharp.Compiler.Services` that Fornax is using would solve this problem. Fornax hasn't been updated in a while, and FCS version is fairly outdated - and as such it may have outdated assumptions about F# installation path (for example - `C:\Program Files (x86)\Microsoft SDKs\F#\10.1` has been introduced in one of the recent VS 2017 updates)

**Comment by drk-mtr** [comment]
Thanks very much for pointing me in the right direction, especially as this is 'user error' rather than a bug in your code!

Just in case anyone else finds this, here's what I did step by step (on Windows):

- Cloned the repository
- Updated the `paket.dependencies` file for the following packages to have the version numbers below:
	`nuget FSharp.Core 4.1.18`
	`nuget FSharp.Compiler.Service 23.0.3`
- Ran `paket install`
- Installed the Sass compiler using `npm install -g sass`
- Ran `build.cmd` in the root of the solution using `.\build`
- Added the `temp\release` folder to my PATH environment variable

Just let me know if you want me to submit a pull request with an updated `paket.dependencies` (with explicit version numbers for FSharp.Core & FSharp.Compiler.Service) and corresponding `paket.lock`.

**Comment by Krzysztof-Cieslak** [comment]
PR welcomed :-)

**Comment by Krzysztof-Cieslak** [comment]
This should be fixed in 0.2
